packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -20,6 +20,10 @@ import { Servers } from '../models/server'
 
 // ===================================================================
 
+const NO_EVENTS_DELAY = 60000 * 5 // 5min
```

Comment:
This must be configurable: no magic values in the code.

Reply:
Where and how you want it to be configurable ?
Can you give an example in the existing code ?

Reply:
- https://github.com/vatesfr/xen-orchestra/blob/29024888fb8e493d9c35de3de5627254bfc39fea/packages/xo-server/src/xo-mixins/authentication.js#L18
- https://github.com/vatesfr/xen-orchestra/blob/29024888fb8e493d9c35de3de5627254bfc39fea/packages/xo-server/config.toml#L36-L46